### DISCUSSION TITLE
Lesson 7: "forge test" is not working!!

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#370] [General]
When I am writing 'forge test' in the Visual Studio Code terminal, I receive this error. I have written a test for the FundMe project code and there are no error or any fault in the code itself.

```
The application panicked (crashed).
Message:  called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }
Location: cli/src/cmd/forge/build/paths.rs:70

This is a bug. Consider reporting it at https://github.com/foundry-rs/foundry

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
Aborted
```

What is the issue??

**Comment by ChefAharoni** [comment]
Are you CD'd in the correct repository with this project's files?

**Reply by Ifra001** [reply]
I wasn't and I didn't even notice. Thank you for pointing it out but now I have a new error which I will probably have to create a new discussion for.
